### ISSUE TITLE
[usage question] How to add style properties for elements "in drag"

### ISSUE THREAD

**Issue by davidseek** [valor-software/ng2-dragula#935]
- [x] My issue title starts with `[usage question]`
- [x] I have read the [README](https://github.com/valor-software/ng2-dragula), especially the ['classic blunders' section](https://github.com/valor-software/ng2-dragula#classic-blunders)
- [x] I have looked at the [demos](https://valor-software.com/ng2-dragula/) to see if my use case has obvious examples.
- [x] I have searched the [`ng2-dragula` issues](https://github.com/valor-software/ng2-dragula/issues), including closed issues.
- [x] I have browsed [the issues labeled "future reference"](https://github.com/valor-software/ng2-dragula/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22future+reference%22) for problems that have been solved before.

**Use case description (required)**

I have a simple table. Each element has a bottom-border.
Once the drag animation starts I would like to have borders left, right and top as well.
Plus instead of the currently high opacity, I would prefer zero opacity and a shadow.
Basically my idea was something like .item:pseudo {}
Is there any way I can archive this?
I appreciate any help or guidance.

**Can you show a demo, or a picture of what you want to achieve?**

<img width="707" alt="screen shot 2019-01-21 at 12 29 09 pm" src="https://user-images.githubusercontent.com/17073950/51497514-2ffe8080-1d78-11e9-9780-e7d6e6787d10.png">

**Comment by davidseek** [comment]
never-mind found it.
